README.adoc

```diff
@@ -69,14 +69,17 @@ Contributing
 
 Thank you for contributing to this repo. Please adhere to the following guidelines when creating a pull request:
 
-. Make your pull request against the _master_ branch (not develio) if you are contributing to the most recent release (currently
+. Make your pull request against the _master_ branch (not develop) if you are contributing to the most recent release (currently
   Cloud 10). The _master_ branch is protected and cannot be merged without review.
 
 . If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also 
   protected and cannot be merged without review.
 
 . Your contributions can be in ASCIIdoc `.adoc` or DocBook XML `.xml`.
 
+. Commit messages must have line lengths of 80 characters or less. Commit body content is required. The commit title must
+  match regex (^(?!SOC).*\((((bsc#|SOC-)[0-9]+(,\s)?)+|trivial|typo|noref)\)$)
```

Comment:
Please explain in more human-friendly terms:
* Subject line must not start with "SOC"
* At the end of the subject line, there must be a bug reference or explanation for a missing bug reference:
  * bug references: "(bsc#0123456)" or "(SOC-0123456)"
  * multiple bug references: "(bsc#0123456)" or "(SOC-0123456)"
  * the values "(trivial)", "(typo)", or "(noref)", no combinations
    allowed

Reply:
@rsalevsky One thing I am wondering -- aren't you using SCRD- for Jira references?
(Also Jira references in the rest of SUSE are actually written with jsc# infront, i.e. jsc#PRJ-0123.)

Reply:
changed. Please check though, not sure about multiple references. Debated about quotes or not.